milestone2.ts

```diff
@@ -82,33 +83,51 @@ const userData: Volunteer[] = [
 /* Define a new lambda function that finds the average age of the users in the data. 
   Hint: user data is stored in the userData object above. 
 */
+// type GetNumber = (data: Volunteer[]) => number;
+// let findAverage: GetNumber; // Define lambda function here
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
-
-//console.log(findAverage(userData));
+let findAverage: GetNumber = (data) => {
+  // Code here
+  var count:number = 0;
```

Comment:
If we are defining a variable that we will only use in this function then use the let keyword so that it can't be accessed elsewhere. Remember that var isn't best practice. Also, the number typing here is a bit unnecessary given that typescript will infer the type once assigned to 0.